site/docs/axis.md

```diff
@@ -84,8 +84,8 @@ To provide themes for all axes, the axis config `config: {axis: {...}}` can cont
 
 ### Ticks
 
-{% include table.html props="tickColor,labelColor,labelFont,labelFontSize,labelLimit,tickWidth" source="AxisConfig" %}
+{% include table.html props="tickColor,tickRound,labelColor,labelFont,labelFontSize,labelLimit,tickWidth" source="AxisConfig" %}
```

Comment:
@domoritz - wonder if you think we should split this?  I think spliting might make more sense for users.